aws-redis-cluster.yml

```diff
@@ -90,6 +85,10 @@ provision:
     details: Comma delimited list of security group ID's for instance
     default: ""
   computed_inputs:
+  - name: cache_size
+    default: -1
+    overwrite: false
+    type: integer
```

Comment:
question: this value will be added to the binding result. Wouldn't be enough by adding `-1` as default value in the definition of the property? Something escapes me that I have not contemplated? Have you seen an element I’m not able to understand?

Reply:
I wasn't able to find any `plan_inputs` with a default value.
I tried adding a default value to `cache_size` directly but integration-tests seemed to suggest that adding defaults in `plan_inputs` is not supported, since `cache_size` always arrived empty to integration tests (when not specified explicitly).

For that reason, I thought adding it as a computed value without overwrite could work as a workaround.
Let me know if there is a better way to represent a default value for a plan property.

Reply:
Removed in commit: https://github.com/cloudfoundry/csb-brokerpak-aws/pull/774/commits/e1b98c701d764d72ef155faf667adf7868ceb771